.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
   docker-rust:
     docker:
       - image: cimg/rust:1.72.0
-    resource_class: small
+    resource_class: medium
```

Comment:
We do not need to bump this

Reply:
Oops mistake, didn't want to bump this one.